src/databricks/labs/pytester/fixtures/catalog.py

```diff
@@ -119,7 +119,7 @@ def create(  # pylint: disable=too-many-locals,too-many-arguments,too-many-state
             catalog_name = schema.catalog_name
             schema_name = schema.name
         if name is None:
-            name = f"ucx_T{make_random(4)}".lower()
+            name = f"dummy_T{make_random(4)}".lower()
```

Comment:
I want to suggest two changes to the names:
1. Replace `dummy` with `test`. IMO it is more accurate as these are resources created for test fixtures
2. Write out the capital letter: `dummy_T<random>` -> `dummy_table_<random>`. I prefer the explicitness. I have used these letters during debugging to know what kind of resources is created. New users will not eaisly recognize this capital letter to mean anything